### PR TITLE
added some new words

### PR DESCRIPTION
**Pull request by Wangluyao0513**
I want to merge

### CODE REVIEW

File: new_file/new_file2.rtf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf600
+{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\paperw11900\paperh16840\margl1440\margr1440\vieww10800\viewh8400\viewkind0
+\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 New file}
```
Comment (fr-kazuhiro-ike) [inline new_file/new_file2.rtf]: I think we should add more text like "I love you so much". 
How do you think ?

Comment (Wangluyao0513) [reply new_file/new_file2.rtf]: I love you so so so so much!

**Review by fr-kazuhiro-ike — APPROVED** [review]
LGTM

**Comment by Wangluyao0513** [comment]
LGTM

**Pull request merged**